.github/workflows/ci-development.yml

```diff
@@ -79,7 +79,7 @@ jobs:
     uses: ./.github/workflows/upgrade-test.yml
     secrets: inherit
     with:
-      run-job: false
+      run-job: true
```

Comment:
Reminder to remove this once we're happy it all passes

Reply:
Passes: https://github.com/chainflip-io/chainflip-backend/actions/runs/10161416189/job/28100173256?pr=5063 so yeah we can remove now